### PR TITLE
Project estimation

### PR DESCRIPTION
**Pull request by amihaiemil**
PR for issue #1095 .
4th project re-estimation

### CODE REVIEW

**Comment by karato** [comment]
@amihaiemil Thanks, I will find someone to review this PR soon

**Comment by karato** [comment]
@krzyk do this review

**Comment by krzyk** [comment]
@amihaiemil Generally you should provide a description of the changes in the PR description, not information that it is a PR :)

File: est/004.est [hunk 1]
```diff
@@ -0,0 +1,24 @@
+date: 13-8-2016
```
Comment (krzyk) [inline est/004.est]: @amihaiemil we generally follow the rule that days and months are written using two digits, so 113-8-20161 should be 113-08-20161

File: est/004.est [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+  3: UI fixes (colors, broken actions etc)
+  4: Bug fixes and refactorings (fix reported issues)
+  5: Unit/Integration testing
+  6: Infrastructure fixes (CI and build related fixes)
+
+champions:
+  1:
+    worst-case: 32
+    best-case: 16
+    most-likely: 24
```
Comment (krzyk) [inline est/004.est]: @amihaiemil here champions should be sorted from descending using `most-likely` field

**Comment by amihaiemil** [comment]
@krzyk done

File: est/004.est [hunk 3]
```diff
@@ -0,0 +1,24 @@
+date: 13-8-2016
+author: Mihai Andronache
+method: champions.pert
+scope:
+  1: Database layer changes
```
Comment (krzyk) [inline est/004.est]: @amihaiemil what do you mean by changes here? Estimation should be done from the project start to the end (at least what we see as end)

**Comment by krzyk** [comment]
@amihaiemil added one more comment

File: est/004.est [hunk 4]
```diff
@@ -0,0 +1,24 @@
+date: 13-08-2016
+author: Mihai Andronache
+method: champions.pert
+scope:
+  1: Database layer changes
```
Comment (krzyk) [inline est/004.est]: @amihaiemil what do you mean by changes here? Estimation should be done from the project start to the end (at least what we see as end)

Comment (amihaiemil) [inline est/004.est]: @krzyk "changes" means the current tasks which are with regards to the DB Layer.. so "Database layer changes" is about all the tasks that are now opened and target that layer. Just "Database layer" seemed to vague to me :D

Comment (krzyk) [inline est/004.est]: @amihaiemil OK

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by karato** [comment]
@ypshenychka review this ticket please, for compliance with our [quality rules](http://at.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor deploy

**Comment by ypshenychka** [comment]
@karato Quality is good.

**Comment by karato** [comment]
> @karato Quality is good.

@ypshenychka thank you, good quality is always good :)

**Comment by karato** [comment]
@krzyk added 10 mins to @ypshenychka, for QA review, payment code is `96668370`

quality is good here, 10 mins added to @dmzaytsev (our architect) in transaction 96668388

thanks, paid, **23 mins** to your account, payment ID is `96668411`, task took 8 hours and 16 mins to complete

review comments (c=8) added as a bonus

+23 added to your rating, current score is: [+1264](http://www.netbout.com/b/35035?open=rating)

**Pull request merged**